html2asketch/nodeTreeToSketchGroup.js

```diff
@@ -16,6 +16,22 @@ export default function nodeTreeToSketchGroup(node, options) {
     // Recursively collect child groups for child nodes
     Array.from(node.children)
       .filter(node => isNodeVisible(node))
+      // sort the children by computed z-index so that nodes with lower z-indexes are added
+      // to the group first, "beneath" those with higher z-indexes
+      .sort(function(a, b) {
```

Comment:
```suggestion
      .sort((a, b) => {
```

For consistent styling I'd use a fat arrow here.